content/news/2018-11-14-community-fund-locks-in.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: news
+generated_by_cms: true
+title: Community Fund Locks-in
+author: John Darby
+date: '2018-11-14T17:05:48+13:00'
+feature_image: /images/uploads/communityfund-lockin.png
+categories:
+  - Nav News
+---
+The big news from the past week in the world of NavCoin is that NavCoin Core 4.4.0 has locked in, activating the Community Fund and the doubling of the network contribution to the Fund from 250,000 to 500,000 NAV per year. That means that the world’s first fully decentralised Community Fund is almost ready to go! Once it activates, there will be nothing holding you back from submitting your ideas and applying to have them funded by the network. 
```

Comment:
I would update this text because it doesn't read right. NavCoin Core 4.4.0 hasn't locked in, but the soft forks included in NavCoin Core 4.4.0 have locked in.